_examples/card/raw-html-card-front.html

```diff
@@ -2,6 +2,8 @@
 <head>
   <link href="../../_global/css/global.css" rel="stylesheet" />
   <link href="../../_global/css/card.css" rel="stylesheet" />
+  <script type="text/javascript" src="../../_global/js/general.js"></script>
+  <script type="text/javascript" src="../../_global/js/card.js"></script>  
```

Comment:
Are these additional includes required?

Reply:
the {icon} shortcuts were not working until I added these